### PR TITLE
Fixes #16024 - escape labels of form fields

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: test/unit/helpers/form_helper_test.rb [hunk 1]
```diff
@@ -51,4 +51,8 @@ class FormHelperTest < ActionView::TestCase
       end
     end
   end
+
+  def h(input)
```
Comment (dLobatog) [inline test/unit/helpers/form_helper_test.rb]: What does this do?

Comment (ares) [inline test/unit/helpers/form_helper_test.rb]: tries to fool IMHO useless and fragile unit tests of some helpers

Comment (ares) [inline test/unit/helpers/form_helper_test.rb]: I've added a comment and put the same method to other test that was failing on the same error

File: app/helpers/form_helper.rb [hunk 2]
```diff
@@ -284,7 +284,7 @@ def add_label options, f, attr
     label = options[:label] == :none ? '' : options.delete(:label)
     label ||= ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
         s_(clazz.gettext_translation_for_attribute_name attr)) if f
-    label = label.present? ? label_tag(attr, "#{label}#{required_mark}".html_safe, :class => label_size + " control-label") : ''
+    label = label.present? ? label_tag(attr, "#{h(label)}#{required_mark}".html_safe, :class => label_size + " control-label") : ''
```
Comment (tbrisker) [inline app/helpers/form_helper.rb]: Just removing the `.html_safe` should do the same trick w/o needing to workaround getting the ERB helper in the tests.

File: vendor/assets/stylesheets/jquery.ui.spinner_custom.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+//span.ui-spinner {
```
Comment (tbrisker) [inline vendor/assets/stylesheets/jquery.ui.spinner_custom.scss]: guessing this was added by mistake to the commit?

Comment (ares) [inline vendor/assets/stylesheets/jquery.ui.spinner_custom.scss]: oops

**Comment by ares** [comment]
thanks for the quick look @tbrisker, I updated the PR

**Comment by dLobatog** [comment]
Looks good, I looked for various `:label =>` in the code, none of them needed html_safe as far as I saw, so ready to merge if Jenkins agrees

**Comment by dLobatog** [comment]
Merged as 0f35fe14acf0d0d3b55e9337bc5e2b9640ff2372, thanks @ares!

**Pull request closed without merging**